src/apps/mean-finance/optimism/mean-finance.balance-fetcher.ts

```diff
@@ -54,17 +57,26 @@ export class OptimismMeanFinanceBalanceFetcher implements BalanceFetcher {
       }
       if (to) {
         to.network = network;
-        tokens.push(drillBalance(to, toWithdraw));
+        tokens.push({
+          ...drillBalance(to, toWithdraw),
+          metaType: MetaType.CLAIMABLE,
+        });
         images = [
           ...images,
           ...getImagesFromToken(to),
         ];
       }
 
       const balanceUSD = sumBy(tokens, t => t.balanceUSD);
+      const swapIntervalAdverb = STRING_SWAP_INTERVALS[swapInterval].adverb;
+      let label = '';
 
-      const label = `Swapping ${from?.symbol} to ${to?.symbol}`;
-      const secondaryLabel = parseInt(remainingSwaps, 10) && STRING_SWAP_INTERVALS[swapInterval] ? `${STRING_SWAP_INTERVALS[swapInterval](remainingSwaps)} left` : 'Position finished';
+      if (remainingSwaps > 0) {
+        label = `Swapping ${rate} ${from?.symbol || dcaPosition.from.symbol} ${swapIntervalAdverb} to ${to?.symbol || dcaPosition.from.symbol}`;
```

Comment:
Use `rate.toFixed(3)` (or some other number of decimals) to keep the label _relatively_ tidy